diff --git a/node_modules/perfect-scrollbar/dist/perfect-scrollbar.esm.js b/node_modules/perfect-scrollbar/dist/perfect-scrollbar.esm.js
index d54e121..973db07 100644
--- a/node_modules/perfect-scrollbar/dist/perfect-scrollbar.esm.js
+++ b/node_modules/perfect-scrollbar/dist/perfect-scrollbar.esm.js
@@ -4,6 +4,17 @@
  * Licensed under MIT
  */
 
+var supportsPassive = false;
+try {
+  var opts = Object.defineProperty({}, 'passive', {
+    get: function() {
+      supportsPassive = true;
+    }
+  });
+  window.addEventListener('testPassive', null, opts);
+  window.removeEventListener('testPassive', null, opts);
+} catch (e) {}
+
 function get(element) {
   return getComputedStyle(element);
 }
@@ -111,7 +122,7 @@ EventElement.prototype.bind = function bind (eventName, handler) {
     this.handlers[eventName] = [];
   }
   this.handlers[eventName].push(handler);
-  this.element.addEventListener(eventName, handler, false);
+  this.element.addEventListener(eventName, handler, supportsPassive ? { passive: true } : false);
 };
 
 EventElement.prototype.unbind = function unbind (eventName, target) {
@@ -565,11 +576,11 @@ function bindMouseScrollHandler(
         (i[contentDimension] - i[containerDimension]) / (i[railDimension] - i[scrollbarDimension]);
 
       if (!e.touches) {
-        document.addEventListener('mousemove', moveHandler);
-        document.addEventListener('mouseup', endHandler);
+        document.addEventListener('mousemove', moveHandler, supportsPassive ? { passive: true } : false);
+        document.addEventListener('mouseup', endHandler, supportsPassive ? { passive: true } : false);
       } else {
         document.addEventListener('touchmove', moveHandler, { passive: false });
-        document.addEventListener('touchend', endHandler);
+        document.addEventListener('touchend', endHandler, supportsPassive ? { passive: true } : false);
       }
 
       i[scrollbarRail].classList.add(cls.state.clicking);
@@ -581,8 +592,8 @@ function bindMouseScrollHandler(
     }
   }
 
-  i[scrollbarAxis].addEventListener('mousedown', bindMoves);
-  i[scrollbarAxis].addEventListener('touchstart', bindMoves);
+  i[scrollbarAxis].addEventListener('mousedown', bindMoves, supportsPassive ? { passive: true } : false);
+  i[scrollbarAxis].addEventListener('touchstart', bindMoves, supportsPassive ? { passive: true } : false);
 }
 
 /* eslint-disable */
