diff --git a/node_modules/vue3-perfect-scrollbar/dist/vue3-perfect-scrollbar.js b/node_modules/vue3-perfect-scrollbar/dist/vue3-perfect-scrollbar.js
index fa7d3a6..c8e77dd 100644
--- a/node_modules/vue3-perfect-scrollbar/dist/vue3-perfect-scrollbar.js
+++ b/node_modules/vue3-perfect-scrollbar/dist/vue3-perfect-scrollbar.js
@@ -48,7 +48,7 @@ const C = /* @__PURE__ */ g({
       var t;
       (t = s.value) != null && t.element && Object.entries(x).forEach(([f, d]) => {
         var m, h;
-        r ? (m = s.value) == null || m.element.addEventListener(f, d) : (h = s.value) == null || h.element.removeEventListener(f, d);
+        r ? (m = s.value) == null || m.element.addEventListener(f, d, { passive: true }) : (h = s.value) == null || h.element.removeEventListener(f, d);
       });
     }
     return n({
